lib/Echidna/Transaction.hs

```diff
@@ -253,7 +252,13 @@ shrinkTx tx'@(Tx c _ _ _ gp (C _ v) (C _ t, C _ b)) = let
 -- | Lift an action in the context of a component of some 'MonadState' to an action in the
 -- 'MonadState' itself.
 liftSH :: (MonadState a m, Has b a) => State b x -> m x
-liftSH = S.state . runState . zoom hasLens
+liftSH = stateST . runState . zoom hasLens
+  -- This is the default state function written in terms of get and set:
+  where stateST :: MonadState s m => (s -> (a, s)) -> m a
+        stateST f = do s <- get
```

Comment:
put this on a newline and indent 2 spaces

Reply:
Sorry, I don't get it. Should the `where` be in the same line as the `liftSH` and the `stateST` start in a newline with indent 2?
```
liftSH = stateST . runState . zoom hasLens where
  -- This is the default state function written in terms of get and set:
  stateST :: MonadState s m => (s -> (a, s)) -> m a
  stateST f = do s <- get
```

Reply:
```haskell
where stateST f = do
  s <- get
  -- ...
```